### PR TITLE
CDRIVER-4009

### PR DESCRIPTION
**Pull request by rcsanchez97**
Full patch build: https://spruce.mongodb.com/version/60d25b52d1fe073c6e1c7624/tasks?limit=100&page=0&sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

### CODE REVIEW

File: src/libmongoc/tests/test-mongoc-collection.c [hunk 1]
```diff
@@ -2600,7 +2600,7 @@ test_estimated_document_count (void)
       collection, tmp_bson ("{'limit': 2, 'skip': 1}"), NULL, &reply, &error);
 
    request = mock_server_receives_msg (
-      server, 0, tmp_bson ("{'count': 'coll', 'limit': 2, 'skip': 1}"));
+      server, 0, tmp_bson ("{'aggregate': 'coll', 'limit': 2, 'skip': 1}"));
```
Comment (benjirewis) [inline src/libmongoc/tests/test-mongoc-collection.c:2603]: Thanks for fixing this on the fly!

**Review by benjirewis — APPROVED** [review]
LGTM as long as those `debug-compile` Windows task failures are unrelated.

**Comment by rcsanchez97** [comment]
The full patch build I linked in the initial summary has only one
failure that is not also present on the base commit, the
`rpm-package-build` task.  That failure is unrelated to these changes.
The other failures you see in the `debug-compile` tasks are not in the
full patch build I submitted, so the are most likely transitory.

**Review by kevinAlbs — APPROVED** [review]
LGTM, thanks for the additional test fixes!

**Comment by rcsanchez97** [comment]
Pushed directly on `master` as:

* d58b2d83d CDRIVER-3905 expect coll stats in estimated document count test
* 1bdfd8fcd CDRIVER-4009 bump wire version for 5.0

**Pull request closed without merging**